examples/plot_roc.py

```diff
@@ -2,7 +2,7 @@
 Plot ROC (Receiver operating characteristic)
```

Comment:
This example could be run in a unit test?

Reply:
The examples are all run when the docs are built. And if they fail, it will be indicated in the PR. But I think we could still have a unit test that just uses the code.

Reply:
That's fine then!